benchmark.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const path = require('path');
+const Benchmark = require('benchmark');
+const makeDir = require('make-dir');
+const tempy = require('tempy');
+const del = require('.');
+
+const suite = new Benchmark.Suite('concurrency');
+
+const tempDir = tempy.directory();
+
+const fixtures = Array.from({length: 2000}, (x, index) => {
+	return path.resolve(tempDir, (index + 1).toString());
+});
+
+function createFixtures() {
+	for (const fixture of fixtures) {
+		makeDir.sync(path.resolve(tempDir, fixture));
+	}
+}
+
+const concurrencies = [1, 3, 5, 10, 15, 20, 50, 100, Infinity];
+
+for (const concurrency of concurrencies) {
+	const name = `concurrency: ${concurrency.toString()}`;
+
+	suite.add({
+		name,
+		defer: true,
+		fn(deferred) {
+			// Can't use setup because it isn't called after every defer
+			// https://github.com/bestiejs/benchmark.js/issues/136
```

Comment:
The referenced issue is closed now.

Reply:
It is just a reference to why setup can't be used. Want me to remove it?